newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -45,6 +46,11 @@ fun Routing.createSongRoutes() {
                     val walletAddress = userRepository.get(song.ownerId!!).walletAddress!!
                     respond(songRepository.refundMintingPayment(songId, walletAddress))
                 }
+                post("reprocess/{mintingStatus}") {
```

Comment:
@wlara I rolled back the Exposed update. Too many changes to how arrays/list column types work and I couldn't get our tests to succeed. You'll have to take a look at that one since you wrote the original array helper code.